### PR TITLE
Use Python scripts instead of manual admin search

### PR DESCRIPTION
**Pull request by OmarIthawi**
This would take less time now and more straight forward to solve.

I'm not sure why we won't let the system does it automatically?

At least, I think we should have it somewhere for CSMs to fix.

Just tried it for 3 users and it worked fine.

### CODE REVIEW

**Review by thraxil — APPROVED** [review]

File: docs/fix_tahoe_user.md [hunk 1]
```diff
@@ -6,67 +6,96 @@
...
+    try:
+        access = AccessToken.objects.get(user=u, client=client)
+    except AccessToken.DoesNotExist:
+        access = AccessToken.objects.create(
+            user=u,
+            client=client,
+            expires=now() + timedelta(days=366),
+        )
+        print 'New AccessToken created'
+    print 'AccessToken:', access
```
Comment (thraxil) [inline docs/fix_tahoe_user.md]:
```suggestion
    print('AccessToken:', access)
```

File: docs/fix_tahoe_user.md [hunk 2]
```diff
@@ -6,67 +6,96 @@
...
+    UserSiteMapping.objects.get_or_create(user=u, site=site)
+    try:
+        access = AccessToken.objects.get(user=u, client=client)
+    except AccessToken.DoesNotExist:
+        access = AccessToken.objects.create(
+            user=u,
+            client=client,
+            expires=now() + timedelta(days=366),
+        )
+        print 'New AccessToken created'
```
Comment (thraxil) [inline docs/fix_tahoe_user.md]:
```suggestion
        print('New AccessToken created')
```

File: docs/fix_tahoe_user.md [hunk 3]
```diff
@@ -6,67 +6,96 @@
...
+    print 'AccessToken:', access
+    try:
+        refresh = RefreshToken.objects.get(user=u, access_token=access, client=client)
+    except RefreshToken.DoesNotExist:
+        refresh = RefreshToken.objects.create(
+            user=u,
+            client=client,
+            access_token=access,
+        )
+        print 'New RefreshToken created'
```
Comment (thraxil) [inline docs/fix_tahoe_user.md]:
```suggestion
        print('New RefreshToken created')
```

File: docs/fix_tahoe_user.md [hunk 4]
```diff
@@ -6,67 +6,96 @@
...
+    print 'RefreshToken:', refresh
```
Comment (thraxil) [inline docs/fix_tahoe_user.md]:
```suggestion
    print('RefreshToken:', refresh)
```

**Comment by OmarIthawi** [comment]
Thanks @thraxil! 

@melvinsoft, should we just merge the two functions in a single one? Or the first one (`ensure_tokens`) is needed for normal users, what do you think?

**Comment by melvinsoft** [comment]
@OmarIthawi It's gonna be closed, but approving so it's out of my list.

**Comment by OmarIthawi** [comment]
That's the best thing ever @melvinsoft :laughing: 

You can always untag yourself from the review list :dancer:

**Pull request closed without merging**